<!DOCTYPE html>
<html>
<head>
<style>
#DB {
  font-family: Arial, Helvetica, sans-serif;
  border-collapse: collapse;
  width: 50%;
}
  

#DB td, #DB th {
  border: 1px solid #ddd;
  padding: 8px;
}

#DB tr:nth-child(even){background-color: #f2f2f2;}

#DB tr:hover {background-color: #ddd;}

#DB th {
  padding-top: 10px;
  padding-bottom: 10px;
  text-align: left;
  background-color: #4CAF50;
  color: white;
}
 
</style>
</head>
<body>

<table id="DB">
  <tr>
    <th>DB Name</th>
    <th>Definition</th>
    <th>Available search</th>
    <th>Applications</th>
    <th>Available GEMs,Pathways,Proteins</th>
    <th>Guide</th>
    <th>Accessibility</th>
    <th>URL</th>
  </tr>
  <tr>
    <td>MetaCyc</td>
    <td>MetaCyc -- index the metabolism world by preserving a representative sample of each pathway which is experimentally explained. </td>
    <td>Pathways,enzymes, reactions,metabolites</td>
    <td>Predict sequenced gene metabolic pathways, support enzyme database metabolic engineering, facilitates studies into metabolomics </td>
    <td>Contains 2859 pathways from 3185 different organisms</td>
    <td>https://metacyc.org/MetaCycUserGuide.shtml</td>
    <td>Through MetaCyc.org, or by downloading MetaCyc data files. Add MetaCyc to the browser search bar for instant searching</td>
    <td>https://metacyc.org/</td>
  </tr>
  <tr>
    <td>BIGG Model</td>
    <td>Combines a collection of starred identifiers (BiGG IDs) into a single pool of over 70 publicated metabolic genome-scale networks. </td>
    <td>Browse the content of genome-scale models (metabolites, reaction, gene, or organism).  </td>
    <td>Map the genes to the NCBI annotations, and various external databases are linked to metabolites (KEGG, PubChem, and many more).
      Improve the quality of annotations and standardization in the model, without modifying the content of the reaction.</td>
    <td>The current knowledge base includes 108 GEMs metabolic knowledge</td>
    <td>https://github.com/SBRG/bigg_models/releases</td>
    <td>BiGG can be browsed on the URL website or accessed using a web API</td>
    <td>http://bigg.ucsd.edu/</td>
  </tr>
<td>KEGG</td>
    <td>Base of data enabling users to understand the high-level biological system functions and resources based on the molecular-basis knowledge (genome sequencing datasets) and other high-performance experimental technology.</td>
    <td>Pathway, modules, KO functional orthologs, Genomes, Genes and proteins, Biochemical reactions and Enzyme nomenclature </td>
    <td>Connecting genomes to metabolic pathways using the EC number</td>
    <td>Chemical information - Contains 548 (851, 821) pathways, 11,632 reactions, and 7,890 enzymes, from 680 eukaryotes, 6623 bacteria, 363 archaea (updated: June 2021)</td>
    <td>https://www.genome.jp/kegg/kegg1a.html</td>
    <td>KEGG API is provided for academic use by academic users. The KEGG FTP site for academic users</td>
    <td>https://www.kegg.jp/kegg/</td>
  </tr>
<td>ModelSEED</td>
    <td>Biochemical database designed for standardisation and data integration assistance. This database is incorporated and unified from many major external sources including MetaCyc,KEGG and BiGG.</td>
    <td>GUI consists of a complete column-searchable compound and reaction table. It offers a more comprehensive view of such individuals with compound and reaction landed pages. </td>
    <td>Compares and incorporates annotations from multiple resources and databases. Describes redundancies and thermodynamics computing properties. </td>
<td> ModelSEED now includes 33,978 compounds and 36,645 reactions.</td>
    <td>https://github.com/ModelSEED/ModelSEEDDatabase</td>
    <td>ModelSEED data can be accessed through web interfaces for manual searching or programmatic REST APIs for public access to data in the Apache Solr (https://lucene.apache.org/solr/)
     database. </td>
    <td>https://modelseed.org</td>
  </tr>
<td>MEMOSys 2.0</td>
    <td>The MEtabolic MOdel Research&Developement System (MEMOSys) is a polyvalent database to maintain, store and build GEMs.</td>
    <td>Searches for Model ID, organisms, and taxonomy.</td>
    <td> Build model hierarchies whose distribution changes automatically. Apply additional annotations, which can be derived directly from the (SBML) format, to all stored components. </td>
    <td> The updated database now includes 20 publicly accessible versions, which can be quickly exported into structured formats for further study.</td>
    <td>https://memosys.i-med.ac.at/MEMOSys/help.seam</td>
    <td>The web-based biological computer database stores all of the properties of a metabolic model in the web site.MEMOSys 2.0. as a completely configured device ready-to-use virtual machine image. </td>
    <td>https://memosys.i-med.ac.at/</td>

    </tr>
<td>TCDB</td>
    <td>A Web open, curated and relation-based database containing sequence,
 classification, structural, functional and evolutionary information on 
transport systems of diverse live species is the Transporter 
Classification Database (TCDB). </td>
<td>Search by sequence 
or by TC number and full-text searching</td>
<td>The database TCDB contains information on the Transporter Classification
  (TC) system, which is a categorization method for membrane 
  transport proteins. The TC method is similar to the Enzyme Commission (EC)
  approach for enzyme categorization, because of how it includes both 
  functional and phylogeny information.</td>
    <td>  The database contains 20892 protein sequences
 These proteins are classified into 1565 transporter families based on the TC-system</td>
     <td>https://www.tcdb.org/browse.php</td>
    <td>Database of
the transport classification

TCDB is the Online web resource
(http://www.tcdb.org) that is freely available.  </td>
    <td>https://www.tcdb.org/</td>

  </tr>
</table>

</body>
</html>
