<!DOCTYPE html>
<html>
<head>
<style>
#DB {
  font-family: Arial, Helvetica, sans-serif;
  border-collapse: collapse;
  width: 100%;
}
  
#DB{
zoom: 80%;
}  

#DB td, #DB th {
  border: 1px solid #ddd;
  padding: 8px;
}

#DB tr:nth-child(even){background-color: #f2f2f2;}

#DB tr:hover {background-color: #ddd;}

#DB th {
  padding-top: 12px;
  padding-bottom: 12px;
  text-align: left;
  background-color: #4CAF50;
  color: white;
}
</style>
</head>
<body>

<table id="DB">
  <tr>
    <th>DB Name</th>
    <th>Definition</th>
    <th>Available search</th>
    <th>Applications</th>
    <th>Available GEMs,Pathways,Proteins</th>
    <th>Guide</th>
    <th>Accessibility</th>
    <th>URL</th>
  </tr>
  <tr>
    <td>MetaCyc</td>
    <td>MetaCyc has both a primary and a secondary
  metabolism pathway,as well as a related metabolite,
reaction, enzyme and genetic function.
The MetaCyc purpose is to index the metabolism world by
preserving a representative sample of each pathway which
is experimentally explained. </td>
    <td>Search for pathways,enzymes,
reactions,metabolites</td>
    <td>Online
metabolism encyclopaedia

Predict sequenced
gene metabolic pathways

Supporting enzyme
database metabolic engineering

Metabolite database facilitates
studies into metabolomics </td>
    <td>MetaCyc contains 
2859 pathways from 
3185 different 
organisms</td>
    <td>https://metacyc.org/
MetaCycUserGuide.shtml</td>
    <td>MetaCyc data can be accessed in several ways:

    Search for pathways, enzymes, reactions, and metabolites through MetaCyc.org
    Download MetaCyc data files
    Browse a list of all MetaCyc metabolic pathways
    Add MetaCyc to your browser search bar for instant MetaCyc searching</td>
    <td>https://
metacyc.org/</td>
  </tr>
  <tr>
    <td>BIGG Model</td>
    <td>BiGG Models is a knowledgebase of GEMs, it combines a collection of starred identifiers known 
as BiGG IDs into a single pool of over 70 publicated metabolic genome-scale networks.
The BiGG model genes are mapped to the NCBI annotations, and various
external databases are linked to metabolites (KEGG, PubChem, and many more). </td>
    <td>searching for a metabolite, reaction, gene, or organism in the search box,
 browse the content of genome-scale models.  </td>
    <td>Improve the quality of annotations and
standardization in the model,
without modifying the content, parameterization or 
relationships of the reaction, errors are identified,
Gene identifiers linked to NCBI RefSeq genome annotation</td>
    <td>The current
knowledge base includes 108
GEMs metabolic knowledge</td>
    <td>https://github.com/SBRG/bigg_models/releases</td>
    <td>BiGG can be browsed on the URL website
or accessed using a web API</td>
    <td>http://bigg.ucsd.edu/</td>
  </tr>
<td>KEGG</td>
    <td>KEGG is a base of data that enables 
   users to understand the high-level biological 
  system functions and resources, such as the cells,
 organisms and environments by means of knowledge on
 a molecular basis, notably large molecular data sets 
provided by genome sequencing and other high-performance
 experimental technology.</td>
    <td>searching for pathway, modules, KO functional orthologs,
 Genomes, Genes and proteins, Biochemical reactions and
 Enzyme nomenclature
 </td>
    <td> KEGG presented the mapping concept for
connecting genomes to metabolic pathways using the CE number</td>
<td> Chemical information
KEGG COMPOUND  18,773
KEGG PATHWAY  542 (765,460)
KEGG REACTION  11,496
KEGG ENZYME 7,813
KEGG ORTHOLOGY 24,034
KEGG GENOME/organisms
(545 eukaryotes, 6118 bacteria, 339 archaea)
KEGG selected viruses 7,002</td>
    <td>https://www.genome.jp/kegg/kegg1a.html</td>
    <td>KEGG API is provided for academic use by academic users 
The KEGG FTP site for academic users</td>
    <td>https://
www.kegg.jp/kegg/</td>
  </tr>
<td>ModelSEED</td>
    <td>Biochemical ModelSEED database, a straightforward biochemical resource
designed for standardisation and data integration assistance.
This database is incorporated and unified from many major external
sources including  MetaCyc,KEGG and BiGG.</td>
    <td>The Modelseed gui consists of a complete column-searchable compound and
reaction table, including support for aliases from other databases.
The gui also offers a more comprehensive view
of such individuals with compound and reaction landed pages. 
 </td>
    <td> The database include a new biochemical design, 
in order for annotations from multiple 
resources and databases to be compared and incorporate.
Redundancies and thermodynamics
computing properties are described. </td>
<td> ModelSEED now includes
 33,978 compounds and
 36,645 reactions</td>
    <td>https://github.com/ModelSEED/ModelSEEDDatabase</td>
    <td>Two ways to
    access ModelSEED data:
    -ModelSEED biochemistry data web
    interfaces for manual searching.
    -Via programmatic APIs, all data can be loaded
     in a REST API for public access to data in 
     the Apache Solr (https://lucene.apache.org/solr/)
     database. </td>
    <td>https://
modelseed.org</td>

  </tr>
<td>MEMOSys 2.0</td>
    <td>The MEtabolic MOdel Research&Developement System (MEMOSys)
    is a polyvalent database to maintain, store and build
    genome-scale models (GEMs). The database has endured massive 
    updates since its original release and some new enhancements
    are added with the new release</td>
    <td>Search for Model ID,
        organisms,
        taxonomy,</td>
    <td> It allows users
     to build model hierarchies which distribute changes automatically.
     Applying additional annotations, which can be derived directly
     from the  (SBML) format, to all stored components.A significant review of the
     web application now involves a new issue </td>
<td> the updated database now includes 20 publicly accessible versions, which
can be quickly exported into structured formats for further study </td>
    <td>https://memosys.i-med.ac.at/MEMOSys/help.seam</td>
    <td>The web-based biological computer database stores all
 of the properties of a metabolic model in the web site.MEMOSys 2.0. as a completely
configured device ready-to-use virtual machine image. </td>
    <td>https://
memosys.i-med.ac.at/</td>

    </tr>
<td>TCDB</td>
    <td>A Web open, curated and relation-based database containing sequence,
 classification, structural, functional and evolutionary information on 
transport systems of diverse live species is the Transporter 
Classification Database (TCDB). </td>
<td>TCDB</td>
    <td>  The database contains 20892 protein sequences
 These proteins are classified into 1565 transporter families based on the TC-system</td>
     <td>https://www.tcdb.org/browse.php</td>
    <td>Database of
the transport classification

TCDB is the Online web resource
(http://www.tcdb.org) that is freely available.  </td>
    <td>https://www.tcdb.org/</td>

  </tr>
</table>

</body>
</html>
