<!DOCTYPE html>
<html>
<head>
<style>
#DB {
  font-family: Arial, Helvetica, sans-serif;
  border-collapse: collapse;
  width: 50%;
  font-size: 70%;
}
  

#DB td, #DB th {
  border: 1px solid #ddd;
  padding: 8px;
}
  
  .td_size {    
  width:200px; 
  height:50px;
  max-width:200px;
  min-width:200px; 
  max-height:50px; 
  min-height:50px;
  **overflow:hidden;** /*(Optional)This might be useful for some overflow contents*/   
}
  
    .td_s {    
  width:100px; 
  height:50px;
  max-width:200px;
  min-width:200px; 
  max-height:50px; 
  min-height:50px;
  **overflow:hidden;** /*(Optional)This might be useful for some overflow contents*/   
}

#DB tr:nth-child(even){background-color: #f2f2f2;}

#DB tr:hover {background-color: #ddd;}

#DB th {
  padding-top: 10px;
  padding-bottom: 10px;
  text-align: left;
  background-color: #4CAF50;
  color: white;
}
 
</style>
</head>
<body>

<table width="50%" id="DB">
  <tr>
    <th>DB Name</th>
    <th>Scope</th>
    <th>Available search</th>
    <th>Applications</th>
    <th>DB Information</th>
    <th>Guide</th>
    <th>Accessibility</th>
    <th>URL</th>
  </tr>
  <tr>
    <td><strong>MetaCyc</strong></td>
    <td class="td_size">Consists of a database of non-redundant and experimentally elucidated metabolic pathways and enzymes.</td>
    <td class="td_s">Pathways, enzymes, reactions, and metabolites.</td>
    <td class="td_size">Predict sequenced gene metabolic pathways, support enzyme database metabolic engineering, facilitates studies into metabolomics.</td>
    <td class="td_size">Contains 2859 pathways from 3185 different organisms.</td>
    <td width="40%"><a href="https://metacyc.org/MetaCycUserGuide.shtml"><span style="color: grey">https://metacyc.org/MetaCycUserGuide.shtml</span></a></td>
    <td class="td_s">Website and browser search bar for instant searching, or by downloading MetaCyc data files</td>
    <td width="40%"><a href="https://metacyc.org/"><span style="color: grey">https://metacyc.org/</span></a></td>
  </tr>
  <tr>
    <td><strong>BIGG Model</strong></td>
    <td>Combines a collection of stardized identifiers (BiGG IDs) into a single pool of over 70 publicated metabolic genome-scale networks.</td>
    <td>Metabolites, reaction, gene, or organism.</td>
    <td>Map the genes to the NCBI annotations, and various external databases are linked to metabolites (KEGG, PubChem, and many more). Improve the quality of annotations and standardization in the model, without modifying the content of the reaction.</td>
    <td>The current knowledge base includes 108 GEMs metabolic knowledge.</td>
    <td width="40%"><a href="https://github.com/SBRG/bigg_models/releases"><span style="color: grey">https://github.com/SBRG/bigg_models/releases</span></a></td>
    <td>Website or web API</td>
    <td width="40%"><a href="http://bigg.ucsd.edu/"><span style="color: grey">http://bigg.ucsd.edu/</span></a></td>
  </tr>
<td><strong>KEGG</strong></td>
    <td>Enables users to understand the high-level biological system functions and resources based on the molecular-basis knowledge (genome sequencing datasets).</td>
    <td>Pathway, KO functional orthologs, genes and proteins, reactions and enzyme nomenclature.</td>
    <td>Connecting genomes to metabolic pathways using the Enzyme Commission (EC) number.</td>
    <td>Chemical information - Contains 548 (851, 821) pathways, 11,632 reactions, and 7,890 enzymes, from 680 eukaryotes, 6623 bacteria, 363 archaea (updated: June 2021).</td>
    <td width="40%"><a href="https://www.genome.jp/kegg/kegg1a.html"><span style="color: grey">https://www.genome.jp/kegg/kegg1a.html</span></a></td>
    <td>API and KEGG FTP site (academic users)</td>
    <td width="40%"><a href="https://www.kegg.jp/kegg/"><span style="color: grey">https://www.kegg.jp/kegg/</span></a></td>
  </tr>
<td><strong>ModelSEED</strong></td>
    <td>Consists of a biochemical database (incorporated and unified from many major external sources including MetaCyc, KEGG and BiGG) designed for standardisation and data integration assistance.</td>
    <td>Compound and reaction table through GUI.</td>
    <td>Compares and incorporates annotations from multiple resources and databases. Describes redundancies and thermodynamics computing properties. </td>
    <td>ModelSEED now includes 33,978 compounds and 36,645 reactions.</td>
    <td width="40%"><a href="https://github.com/ModelSEED/ModelSEEDDatabase"><span style="color: grey">https://github.com/ModelSEED/ModelSEEDDatabase</span></a></td>
    <td>Web interfaces (manual searching) or programmatic REST APIs (public access to the Apache Solr database)</td>
    <td width="40%"><a href="https://modelseed.org"><span style="color: grey">https://modelseed.org</span></a></td>
  </tr>
<td><strong>MEMOSys 2.0</strong></td>
    <td>Represents a polyvalent database able to maintain, store and build GEMs.</td>
    <td>Model ID, organisms, and taxonomy.</td>
    <td>Builds model hierarchies whose distribution changes automatically. Apply additional annotations, which can be derived directly from the (SBML) format, to all stored components.</td>
    <td>The updated database now includes 20 publicly accessible versions, which can be quickly exported into structured formats for further study.</td>
    <td width="40%"><a href="https://memosys.i-med.ac.at/MEMOSys/help.seam"><span style="color: grey">https://memosys.i-med.ac.at/MEMOSys/help.seam</span></a></td>
    <td>Website and ready-to-use virtual machine image</td>
    <td width="40%"><a href="https://memosys.i-med.ac.at/"><span style="color: grey">https://memosys.i-med.ac.at/</span></a></td>

    </tr>
<td><strong>TCDB</strong></td>
    <td>Consists of a curated database containing sequences, classification, structural, functional and evolutionary information on transport systems of diverse live species. </td>
    <td>By sequence or TC number and full-text searching.</td>
    <td>Categorizes membrane transport proteins, using information on the TC system. The TC method is similar to the EC approach for enzyme categorization, because of how it includes both functional and phylogeny information.</td>
    <td> The database contains 20892 protein sequences classified into 1565 transporter families based on the TC system.</td>
    <td width="40%"><a href="https://www.tcdb.org/browse.php"><span style="color: grey">https://www.tcdb.org/browse.php</span></a></td>
    <td>Website</td>
    <td width="40%"><a href="https://www.tcdb.org/"><span style="color: grey">https://www.tcdb.org/</span></a></td>

  </tr>
</table>

</body>
</html>
